semantic-conventions/syntax.md

```diff
@@ -168,6 +178,26 @@ The following is only valid if `type` is `event`:
   If not specified, the `prefix` is used. If `prefix` is empty (or unspecified),
   `name` is required.
 
+#### Metric Group semantic convention
+
+Metric group inherits all from the base semantic convention, and does not 
+add any additional fields. Its only requirement is that `type` is `metric_group`. 
+
+The metric group semconv is intended to be a group where metric attributes 
+can be defined and then referenced from other `metric` groups. 
```

Comment:
Referenced how?

Reply:
Using `ref`, the same as span attributes do. I've clarified.